src/main/resources/templates/mdBenefitsFlow/mailingAddressNoAddress.html

```diff
@@ -19,8 +19,7 @@
                 <th:block th:replace="~{fragments/inputs/address ::
                                   address(
                                     validate=true,
-                                    inputName='mailingAddress',
-                                    streetAddress2HelpText=#{general.optional}
+                                    inputName='mailingAddress'
```

Comment:
I thought we talked about keeping this (at least as the smaller group)?  Or was there further discussion?

Reply:
There was no further discussion and I was going to bring it up today.